crates/bevy_core_pipeline/src/postprocess/postprocess.wgsl

```diff
@@ -0,0 +1,81 @@
+// Miscellaneous postprocessing effects, currently just chromatic aberration.
+
+#import bevy_core_pipeline::fullscreen_vertex_shader::FullscreenVertexOutput
+
+// See `bevy_core_pipeline::postprocess::ChromaticAberration` for more
+// information on these fields.
+struct PostprocessingSettings {
+    chromatic_aberration_intensity: f32,
+    chromatic_aberration_max_samples: u32,
+    unused_a: u32,
+    unused_b: u32,
+}
+
+// The source framebuffer texture.
+@group(0) @binding(0) var source_texture: texture_2d<f32>;
+// The sampler used to sample the source framebuffer texture.
+@group(0) @binding(1) var source_sampler: sampler;
+// The 1D lookup table for chromatic aberration.
+@group(0) @binding(2) var chromatic_aberration_lut_texture: texture_1d<f32>;
+// The sampler used to sample that lookup table.
+@group(0) @binding(3) var chromatic_aberration_lut_sampler: sampler;
+// The settings supplied by the developer.
+@group(0) @binding(4) var<uniform> settings: PostprocessingSettings;
+
+@fragment
+fn fragment_main(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
+    // Radial chromatic aberration implemented using the *Inside* technique:
+    //
+    // <https://github.com/playdeadgames/publications/blob/master/INSIDE/rendering_inside_gdc2016.pdf>
```

Comment:
This repo doesn't have a license, but I have no idea how pdf slides should be handled in terms of licensing. Does it just count as public domain so we can use it freely?

Reply:
(Not really a blocker, ju$t my own curiosity)

Reply:
The code is entirely original; I'm just using the high-level technique, which isn't copyrightable (IANAL).